CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Fixed maximum value of Pipe999LoopCrossfadeLength and Pipe999ReleaseCrossfadeLength. Now they are 3000 https://github.com/GrandOrgue/grandorgue/issues/1612
```

Comment:
not "Fixed"! It should read:

Increased maximum...

Reply:
> not "Fixed"! It should read:
> 
> Increased maximum...

Should this change will be delayed until 3.13.0 as it is not a bug?

Reply:
If that suits your overall planning. I won't mind anyway.

Reply:
I planned it include to 3.12.3 as I considered this (two strong restriction) as a bug. But if we reclassify it as enchancement, we may not include this change to the bugfix-only version 3.12.3.

So we either write `Fixed` and include this change to 3.12.3 or we write `Increased` and don't include it to 3.12.3.

What woud you decision be?

Reply:
Delay it then. We should get it out fairly quickly anyway.

Reply:
Fixed